gensim/models/ldaseqmodel.py

```diff
@@ -1091,10 +1089,8 @@ def update_obs(self, sstats, totals):
                     w_counts = np.zeros(len(w_counts))
 
                 # TODO: apply lambda function
-                for t in range(0, T):
-                    mean_deriv = mean_deriv_mtx[t]
-                    mean_deriv = self.compute_mean_deriv(w, t, mean_deriv)
-                    mean_deriv_mtx[t] = mean_deriv
+                for t in range(T):
+                    mean_deriv_mtx[t] = self.compute_mean_deriv(w, t, mean_deriv_mtx[t])
```

Comment:
are you sure? is `compute_mean_deriv` don't mutate arguments?

Reply:
It's good point. I've check it and `compute_mean_deriv` changes `mean_deriv_mtx[t]` and returns it =).

Reply:
> I've check it and compute_mean_deriv changes mean_deriv_mtx[t] and returns it =).

Oh jesus ...